webapi/Models/Response/BotResponsePrompt.cs

```diff
@@ -67,16 +62,15 @@ public class BotResponsePrompt
         string documentMemories,
         SemanticDependency<StepwiseThoughtProcess> externalInformation,
         string chatHistory,
-        string systemChatContinuation
-        )
+        ChatHistory metaPromptTemplate
```

Comment:
Same. `ChatHistory` is a type and variable name at the same time. Makes it a bit hard to understand at the first glance.

Reply:
Renamed messages used for meta-prompt template to use `ChatCompletionContextMessages` type and `contextMessages` as variable name where applicable